lib/Tga/SimHash/Fingerprint.php

```diff
@@ -24,19 +24,19 @@ class Fingerprint
     /**
      * @var float
      */
-    protected $decimalValue;
+    protected $value;
```

Comment:
This is a BC break without any added value, could you revert this?

Reply:
"decimalValue" save value as number.
"value" save  value as string.

These are two different variables, so they are called differently.

Reply:
Could you make this property private as well? I don't think it should be exposed anyway.